resources/lang/en/events/lead_ceremony_df.json

```
@@ -128,7 +128,572 @@
+      "stubbornness", "authority"]
+        }
+      ]
+    },
+
+    "ambitious_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["ambitious"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "r_c's dark eyes glisten with the light of a thousand unrealized ambitions. As {PRONOUN/r_c/subject} {VERB/r_c/inflict/inflicts} a life for [virtue] on m_c, r_c hisses those goals into {PRONOUN/m_c/poss} ear.",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunning", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+      "stubbornness", "authority"]
+        }
+      ]
+    },
+  
+  	"bold_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["bold"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "r_c bares {PRONOUN/r_c/poss} teeth and looks like {PRONOUN/r_c/subject} {VERB/r_c/are/is} about to fight m_c. But r_c only presses {PRONOUN/r_c/poss} nose against m_c and snarls a life for [virtue].",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunning", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+      "stubbornness", "authority"]
+        }
+      ]
+    },
+  
+    "calm_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["calm"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "The air falls into an unsettling stillness as r_c, a cat with nefarious reputation, draws near, exuding a deceptive calm. With measured steps, {PRONOUN/r_c/subject} {VERB/r_c/approach/approaches} m_c, extending an offer of a life infused with the virtue of [virtue], to then vanish into the veils of darkness from which {PRONOUN/r_c/subject} emerged.",
+          "virtue": ["ambition", "certainty", "clear judgment", "confidence", "courage", "endurance", "farsightedness", "instincts", "cunningness", "shamelessness", "zealotry", "bloodlust", "authority"]
+        },
+        {
+          "text": "r_c approaches, eerily composed, {PRONOUN/r_c/poss} calmness tinged with a malevolent presence. {PRONOUN/r_c/poss/CAP} cool facade gleams at m_c, as {PRONOUN/r_c/subject} {VERB/r_c/extend/extends} an offer of a life, infused with the virtue of [virtue].",
```

```suggestion
          "text": "Eerily composed, r_c approaches, {PRONOUN/r_c/poss} calmness tinged with a malevolent presence. {PRONOUN/r_c/poss/CAP} cool facade gleams at m_c, as {PRONOUN/r_c/subject} {VERB/r_c/extend/extends} an offer of a life, infused with the virtue of [virtue].",
```
Separating out the two calmness flavours feels less like we're recursively nesting clauses.